modules/backend/formwidgets/repeater/assets/js/repeater.js

```diff
@@ -44,6 +44,7 @@
         this.$el.one('dispose-control', this.proxy(this.dispose))
 
         this.hidePrompt()
+        $(document).on('ajaxSuccess', this.proxy(this.hidePrompt))
```

Comment:
Does this event need to be bound to `document`? Why not bind it more specifically to `this.$el`?